src/aiida/cmdline/commands/cmd_status.py

```
@@ -151,56 +150,64 @@ def verdi_status(print_traceback: bool, no_rmq: bool) -> None:
-        print_status(ServiceStatus.WARNING, 'daemon', str(exception))
-    except DaemonException as exception:
-        print_status(ServiceStatus.ERROR, 'daemon', str(exception))
-    except Exception as exception:
-        message = 'Error getting daemon status'
-        print_status(ServiceStatus.ERROR, 'daemon', message, exception=exception, print_traceback=print_traceback)
-        exit_code = ExitCode.CRITICAL
+    if profile.process_control_backend is None:
+        try:
+            daemon_client = manager.get_daemon_client()
+            is_daemon_running = daemon_client.is_daemon_running
+        except Exception as exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)